### PR TITLE
fix(instagram): assorted fixes

### PR DESCRIPTION
**Pull request by GenShibe**
## 🔧 What does this fix? 🔧
you're all caught up checkbox, story progress bar, dm message timestamp and more lol

## 🗒 Checklist 🗒

- [x] I have read and followed Catppuccin's [contributing guidelines](https://github.com/catppuccin/userstyles/blob/main/docs/CONTRIBUTING.md).
- [x] I have updated the version appropriately in the `==UserStyle==` header of the `catppuccin.user.css` file.

### CODE REVIEW

File: styles/instagram/catppuccin.user.css [hunk 1]
```diff
@@ -143,7 +143,10 @@
       box-shadow: 0 3px 5px -1px fadeout(@dark_color, 20);
       box-sizing: border-box;
     }
-
+    // testing 13
+    ._ac3p {
+      background-color: @accent-color !important;
+    }
```
Comment (uncenter) [inline styles/instagram/catppuccin.user.css:149]: ... testing 13?

Comment (GenShibe) [reply styles/instagram/catppuccin.user.css:149]: oooops forgot to remove that LOL

File: styles/instagram/catppuccin.user.css [hunk 2]
```diff
@@ -58,72 +53,62 @@
     @mantle: @catppuccin[@@lookup][@mantle];
     @crust: @catppuccin[@@lookup][@crust];
     @accent-color: @catppuccin[@@lookup][@@accent];
+    @text-filter: @catppuccin[@@lookup][@text-filter];
+    @dark-color: if(@lookup=latte, @text, @crust);
+    @light-color: if(@lookup=latte, @crust, @text);
```
Comment (uncenter) [inline styles/instagram/catppuccin.user.css:58]:
```suggestion
    @dark-color: if(@lookup = latte, @text, @crust);
    @light-color: if(@lookup = latte, @crust, @text);
```

File: styles/instagram/catppuccin.user.css [hunk 3]
```diff
@@ -135,36 +120,43 @@
...
+    /* The close icon on the dark wrapper */
     svg[aria-label="Close"] {
       color: var(--web-always-white);
     }
     svg[aria-label="Verified"] {
       fill: @accent-color;
     }
 
-    //Share box
+    /*Share box*/
```
Comment (uncenter) [inline styles/instagram/catppuccin.user.css:159]:
```suggestion
    /* Share box */
```

File: styles/instagram/catppuccin.user.css [hunk 4]
```diff
@@ -174,20 +166,18 @@
...
+      color: @text !important;
       &:hover {
-        color: @mantle;
+        color: @crust;
       }
     }
 
-    //all toggle sliders
-
+    /*all toggle sliders*/
```
Comment (uncenter) [inline styles/instagram/catppuccin.user.css:180]:
```suggestion
    /* Toggle sliders */
```

**Review by uncenter — CHANGES_REQUESTED** [review]

File: styles/instagram/catppuccin.user.css [hunk 5]
```diff
@@ -504,17 +491,29 @@
       @mauve 60%,
       @blue 90%
     );
+    img[src="/images/instagram/xig/web/illo-confirm-refresh-light.png"] {
```
Comment (uncenter) [inline styles/instagram/catppuccin.user.css:494]:
```suggestion
	/* "All caught up" icon */
    img[src="/images/instagram/xig/web/illo-confirm-refresh-light.png"] {
```

**Review by isabelroses — APPROVED** [review]

**Review by uncenter — APPROVED** [review]

**Pull request merged**